_pages/clients/java.md

```diff
@@ -261,6 +261,21 @@ This guide used the the first chapter of [Jack London's Burning Daylight](https:
 
 If you are interested in learning more about working with unstructured text data check out our guide [Introduction to Natural Language Processing](https://algorithmia.com/blog/introduction-natural-language-processing-nlp).
 
+{% if site.enterprise %}
+## Publishing Algorithmia Insights
+
+Inference-related metrics (a feature of [Algorithmia Insights](../../algorithmia-enterprise/algorithmia-insights)) can be reported via using the `reportInsights` method of the Algorithmia client.
```

Comment:
```suggestion
Inference-related metrics (a feature of [Algorithmia Insights](../algorithmia-enterprise/insights)) can be reported via using the `reportInsights` method of the Algorithmia client.
```

Reply:
It turns out [product has explicitly asked us to use "Algorithmia Insights" everywhere for branding purposes](https://github.com/algorithmiaio/dev-center/pull/340#pullrequestreview-482276022)!

Reply:
Ah gotcha, in that case `_pages/algorithmia-enterprise/insights.md` will have to be renamed to `_pages/algorithmia-enterprise/algorithmia-insights.md` so it doesn't 404.

Reply:
`../../algorithmia-enterprise/algorithmia-insights` should be `../algorithmia-enterprise/algorithmia-insights`. The double `../` removes `developers` from the path.

Reply:
Thanks Nathan! Changes made!

Reply:
(all tested and working on https://insights1.enthalpy.click/developers/algorithmia-enterprise/algorithmia-insights)